benchmarks/src/main/java/com/esotericsoftware/kryo/benchmarks/io/FastGetObjectMapBenchmark.java

```diff
@@ -0,0 +1,216 @@
+package com.esotericsoftware.kryo.benchmarks.io;
```

Comment:
Please move this benchmark to `com.esotericsoftware.kryo.benchmarks.io` since the benchmark actually tests the maps in the `util` package.